salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -6,33 +6,6 @@ Execute the upgrade prechecks:
     - mods:
       - metalk8s.orchestrate.upgrade.precheck
     - saltenv: {{ saltenv }}
-    - pillar:
```

Comment:
Would it make sense to now add a big fat warning at the beginning of this SLS (in comments) to clarify this state does *not* upgrade a cluster fully, but only does part of the job, pointing at the upgrade script etc.?

Reply:
Btw it does the full upgrade because it call highstate everywhere and upgrade k8s object just it does not follow the "upgrade process" but it this state succeed at the end the cluster is upgraded :smiley: